docs/user_guide/selection/index.rst

```diff
@@ -151,9 +151,3 @@ Click below to find more details on how to use each one of the transformers.
 
 Additional Resources
 --------------------
-
-More details about feature selection can be found in the following resources:
```

Comment:
These lines were not the ones you added, they were there already. Could you please add them back?